source/slang/slang.cpp

```diff
@@ -1965,6 +1974,87 @@ void Module::_addEntryPoint(EntryPoint* entryPoint)
     m_entryPoints.add(entryPoint);
 }
 
+static bool _canExportDeclSymbol(ASTNodeType type)
```

Comment:
The new `static` functions here probably deserve documentation.

Reply:
I might argue for an even more straightforward-but-wordy name for this function. Something like `_canDeclsOfTypeBeExported` to match the semantics of "this function returns `true` if declarations of the given `type` can be exported from a module"